_includes/gha-tutorial.md

```diff
@@ -0,0 +1,178 @@
+This guide walks you through the process of setting up and using Docker GitHub
+Actions for building Docker images, and pushing images to Docker Hub. You will
+complete the following steps:
+
+1. Create a new repository on GitHub.
+2. Define the GitHub Actions workflow.
+3. Running the workflow.
+
+To follow along with this guide, you'll need a Docker ID and a GitHub account.
+
+### Create the repository
+
+Create a GitHub repository and configure the Docker Hub secrets.
+
+1. Create a new GitHub repository using
+   [this template repository](https://github.com/dvdksn/clockbox/generate).
+
+   The repository contains a simple Dockerfile, and nothing else. Feel free to
+   use another repository containing a working Dockerfile if you prefer.
+
+2. Open the repository **Settings**, and go to **Secrets** > **Actions**.
+
+3. Create a new secret named `DOCKER_HUB_USERNAME` and your Docker ID as value.
+
+4. Create a new
+   [Personal Access Token (PAT)](https://docs.docker.com/docker-hub/access-tokens/#create-an-access-token)
+   for Docker Hub. You can name this token `clockboxci`.
+
+5. Add the PAT as a second secret in your GitHub repository, with the name
+   `DOCKER_HUB_ACCESS_TOKEN`.
+
+With your repository created, and secrets configured, you're now ready for
+action!
+
+### Set up the workflow
+
+Set up your GitHub Actions workflow for building and pushing the image to Docker
+Hub.
+
+1. Go to your repository on GitHub and then select the **Actions** tab.
+2. Select **set up a workflow yourself**.
```

Comment:
I actually only saw this after I had selected **New workflow** then it took me to another page where I saw **set up a workflow**. Might just be my set up though?